.travis.install-kafka.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -e
+
```

Comment:
I usually use this blurb:
```
set -o errexit
set -o nounset
set -o pipefail
```

A little more verbose. I think `nounset` is a bashism, so it might not work here.

Reply:
no reason we can't use bash